tests/SendGridHandlerTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Tests;
+
+use GuzzleHttp\Psr7\HttpFactory;
+use Monolog\Logger;
+use MonologHttp\SendGridHandler;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Log\LogLevel;
+
+final class SendGridHandlerTest extends TestCase
+{
+    /**
+     * @var MockObject|ClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var Logger
+     */
+    private $logger;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->httpClient = $this->createMock(ClientInterface::class);
+        $this->logger = new Logger('test');
+        $this->logger->pushHandler(
+            new SendGridHandler(
+                $this->httpClient,
+                new HttpFactory(),
+                'apiuser',
+                'apikey',
+                'from@domain.com',
+                ['to@domain.com'],
+                'There was an error'
+            )
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function createRequest(): void
```

Comment:
```suggestion
    public function testCreateRequest(): void
```

Reply:
We are using a convention from php-cs-fixer